components/esp_lcd_touch_gt911/idf_component.yml

```
@@ -0,0 +1,6 @@
+version: "0.0.1"
```

Applies to all `.yml` files
```suggestion
version: "0.9.0"
```

Let's start with version 0.9.0